src/RTTI/RTTI_GameClock.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "RTTI_TypeIDs.hpp"
+#include <BsCorePrerequisites.h>
+#include <Private/RTTI/BsGameObjectRTTI.h>  // Says private, but bs:f uses this too in their RTTIs
+#include <Reflection/BsRTTIType.h>
+#include <components/GameClock.hpp>
+
+namespace REGoth
+{
+  class RTTI_GameClock
+      : public bs::RTTIType<GameClock, bs::Component, RTTI_GameClock>
+  {
+    BS_BEGIN_RTTI_MEMBERS
+    // TODO: Fill RTTI Members
```

Comment:
Should be eas to add since the class only stores two floats. If you need examples, check the serialization branch